linux/mcompile/keymap/u16.h

```diff
@@ -19,10 +19,10 @@ std::u16string u16string_from_string(std::string const str);
 std::u16string u16string_from_wstring(std::wstring const wstr);
 std::string string_from_u16string(std::u16string const str);
 
-std::wstring u16fmt(const KMX_WCHAR * str);
 void u16sprintf(KMX_WCHAR * dst, const size_t sz, const wchar_t* fmt, ...);
 
 std::wstring  convert_pchar16T_To_wstr(KMX_WCHAR *Name);
+std::wstring  convert_pchar16T_To_wstr(const KMX_WCHAR *Name);
```

Comment:
I can only see one implementation of `convert_pchar16T_To_wstr`, so I think we should get rid of the definition without `const`:

```suggestion
std::wstring  convert_pchar16T_To_wstr(const KMX_WCHAR *Name);
```

Reply:
OK-done